content/index.html

```diff
@@ -6,7 +6,8 @@
     <h1>From metrics to insight</h1>
     <p class="subtitle">Power your metrics and alerting with a leading<br>open-source monitoring solution.</p>
     <p>
-      <a class="btn btn-default btn-lg" href="/docs/introduction/getting_started/" role="button">GET STARTED</a>
+      <a class="btn btn-default btn-lg" href="/docs/introduction/getting_started/" role="button">Get Started</a>
+      <a class="btn btn-default btn-lg" href="/download" role="button">Download</a>
```

Comment:
Could you set `margin: 15px 10px 0 10px;` on `.jumbotron a.btn`? Otherwise the buttons are very tightly glued together.